src/tools/auth0/handlers/emailProvider.ts

```diff
@@ -16,7 +17,8 @@ export default class EmailProviderHandler extends DefaultHandler {
 
   async getType(): Promise<Asset> {
     try {
-      return await this.client.emailProvider.get({ include_fields: true, fields: defaultFields });
+      const { data  } = await this.client.emails.get({ include_fields: true, fields: defaultFields.join(',') });
```

Comment:
This looks like a breaking change but the SDK types `fields` as a string so this doesn't precludes the need to type cast.